package-lock.json

```diff
@@ -1317,7 +1369,7 @@
         },
         "chalk": {
             "version": "1.1.3",
-            "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

Comment:
You see how it's changing a bunch of https urls to http? That's due to an old bug in npm. Make sure you're on an up-to-date version of npm (I'm on 6.13.4) and follow these steps to fix: https://github.com/npm/npm/issues/20106#issuecomment-383770315

Reply:
Updated npm to latest version in the next commit